### PR TITLE
Useful utilities review

### PR DESCRIPTION
**Pull request by oliverexell-bruce**
# Description

Please include a summary of the changes and the related issue. Please also include relevant
motivation and context. List any dependencies that are required for this change.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (update or new)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration.

- [X] Tested in latest Chrome
- [X] Tested in latest Firefox
- [X] npm run build
- [X] npm run preview

### If involving code

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by oliverexell-bruce** [comment]
# Description

Please include a summary of the changes and the related issue. Please also include relevant
motivation and context. List any dependencies that are required for this change.

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (update or new)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration.

- [X] Tested in latest Chrome
- [X] Tested in latest Firefox
- [X] npm run build
- [X] npm run preview

## Checklist

### If involving code

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

File: src/content/docs/guides/Utilities/0-useful-utilities.mdx [hunk 1]
```diff
@@ -374,23 +374,23 @@ SplashKit's [Is Integer](/api/utilities/#is-integer) and [Is Number](/api/utilit
...
-      if ( is_integer(buffer) )
+      if ( is_integer(line) )
         break;
       //If the user input was not a valid whole number, ask them to enter a whole number.
       write_line("Please enter a valid whole number.");
     }
     //Convert the user input to an integer before returning it.
-    return convert_to_integer(buffer);
+    return convert_to_integer(line);
   }
```
Comment (omckeon) [inline src/content/docs/guides/Utilities/0-useful-utilities.mdx:394]: Can you also swap this read_integer code (but keep comments where appropriate) for the one in programmers.guide on this page: https://programmers.guide/book/part-2-organised-code/2-organising-code/1-tour/04-00-utilities/#implementing-the-utilities (to remove the use of an infinite loop)? Thanks!

Comment (oliverexell-bruce) [reply src/content/docs/guides/Utilities/0-useful-utilities.mdx]: I have written, tested, and commited this change.
image

**Comment by breezy-codes** [comment]
I've reviewed all program code and they run correctly as intended and are free of errors. The guide updates are great.
Especially catching the program code error `// Start of "main" code` in the python part of one code. This is why the tutorial review process was set up!
Good work

**Review by breezy-codes — APPROVED** [review]

**Review by NidhishaPahade — APPROVED** [review]
Everything looks good !! Great work

**Review by omckeon — COMMENTED** [review]
Just to let you know that I've added the C# OO tabs to this PR code as it had been missed in the Planner cards.

**Pull request merged**